package.json

```diff
@@ -2,19 +2,34 @@
   "name": "obj-multiplex",
   "version": "1.0.2",
   "description": "",
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "test": "node test"
+    "build": "tsc --project .",
+    "lint": "eslint . --ext ts,js,json",
+    "lint:fix": "yarn lint --fix",
+    "test": "yarn build && node test"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "end-of-stream": "^1.4.0",
+    "end-of-stream": "^1.4.4",
```

Comment:
Question: is there any reason you updated this package? I think it's fine, just wondering.

Reply:
It got updated while trying to fix the `premature closure` error. Even though this change did not have any effect on that error, it did not break anything and it is the latest version of end-of-stream. Should I change it back ??!!!

Reply:
Nah, it's fine. It's just a patch version bump after all!